link_shorterner/src/shortener_link_handler.erl

```diff
@@ -0,0 +1,33 @@
+-module(shortener_link_handler).
+
+-export([init/2]).
+
+init(Req, State) ->
+    Url = get_request_url(Req),
+    ReqMethod = cowboy_req:method(Req),
+    {Status, ShortUrl} = handle_request(ReqMethod, Url),
+    Body = "{url: \"" ++ ShortUrl ++ "\"}",
```

Comment:
`url` is missing quoutes to be valid json

Reply:
please use a json library as jsx